packages/front-end/pages/namespaces.tsx

```diff
@@ -88,7 +88,7 @@ const NamespacesPage: FC = () => {
               <th>Description</th>
               <th>Active experiments</th>
               <th>Percent available</th>
-              {canEdit && <th style={{ width: 30 }}></th>}
+              <th style={{ width: 30 }}></th>
```

Comment:
Removing this simplified things quite a bit. Together with the change on line 120 of `NamespaceTableRow.tsx` ensures that the table is always full width, regardless of if the user has permission to edit or not.